src/main/java/Main.java

```diff
@@ -0,0 +1,49 @@
+public class Main {
+    public static void main(String args[]) {
+        Cat cat = new Cat("Cat");
+        cat.speak();
```

Comment:
Currently in methods hardcoded logic. Please look at paragraph 3 at hw, we should have possibilities to put different values